Makefile

```diff
@@ -8,359 +8,5 @@
-	  -o -name "time-of-build-pretty.log" -o -name "time-of-build-both.log" \) -exec rm -f {} +
-
-plugin-tutorialclean:
-	+$(MAKE) -C $(PLUGINTUTO) clean
-
-# Ensure that every compiled file around has a known source file.
-# This should help preventing weird compilation failures caused by leftover
-# compiled files after deleting or moving some source files.
-
-EXISTINGVO:=$(call find, '*.vo')
-KNOWNVO:=$(patsubst %.v,%.vo,$(call find, '*.v'))
-ALIENVO:=$(filter-out $(KNOWNVO),$(EXISTINGVO))
-
-EXISTINGOBJS:=$(call find, '*.cm[oxia]' -o -name '*.cmxa')
-KNOWNML:=$(EXISTINGML) $(GENMLFILES) $(MLPACKFILES:.mlpack=.ml) \
- $(patsubst %.mlp,%.ml,$(wildcard grammar/*.mlp))
-KNOWNOBJS:=$(KNOWNML:.ml=.cmo) $(KNOWNML:.ml=.cmx) $(KNOWNML:.ml=.cmi) \
- $(MLIFILES:.mli=.cmi) \
- gramlib/.pack/gramlib.cma gramlib/.pack/gramlib.cmxa $(MLLIBFILES:.mllib=.cma) $(MLLIBFILES:.mllib=.cmxa) grammar/grammar.cma
-ALIENOBJS:=$(filter-out $(KNOWNOBJS),$(EXISTINGOBJS))
-
-alienclean:
-	rm -f $(ALIENOBJS) $(ALIENVO)
-
-###########################################################################
-# Continuous Intregration Tests
-###########################################################################
-include Makefile.ci
-
-###########################################################################
-# Emacs tags
-###########################################################################
-
-.PHONY: tags printenv
-
-tags:
-	echo $(filter-out checker/%, $(MLIFILES)) $(filter-out checker/%, $(MLSTATICFILES)) $(MLGFILES) | sort -r | xargs \
-	etags --language=none\
-	      "--regex=/let[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/let[ \t]+rec[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/and[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/type[ \t]+\([^ \t]+\)/\1/" \
-              "--regex=/exception[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/val[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/module[ \t]+\([^ \t]+\)/\1/"
-	echo $(MLGFILES) | sort -r | xargs \
-	etags --append --language=none\
-	      "--regex=/[ \t]*\([^: \t]+\)[ \t]*:/\1/"
-
-checker-tags:
-	echo $(filter-out kernel/%, $(MLIFILES)) $(filter-out kernel/%, $(MLSTATICFILES)) $(MLGFILES) | sort -r | xargs \
-	etags --language=none\
-	      "--regex=/let[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/let[ \t]+rec[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/and[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/type[ \t]+\([^ \t]+\)/\1/" \
-              "--regex=/exception[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/val[ \t]+\([^ \t]+\)/\1/" \
-	      "--regex=/module[ \t]+\([^ \t]+\)/\1/"
-	echo $(MLGFILES) | sort -r | xargs \
-	etags --append --language=none\
-	      "--regex=/[ \t]*\([^: \t]+\)[ \t]*:/\1/"
-
-# Useful to check that the exported variables are within the win32 limits
-
-printenv:
-	@env
-	@echo
-	@echo -n "Maxsize (win32 limit is 8k) : "
-	@env | wc -L
-	@echo -n "Total (win32 limit is 32k) : "
-	@env | wc -m
+# The default build system is make-based one.
+-include Makefile.make
```

Comment:
We could control this via an env variable (COQ_USE_DUNE?) then people could tweak their setup so they don't need to -f to use dune
Not sure if worth the bother

Reply:
Sure why not?

Reply:
Added.

Reply:
But I leave it undocumented for now, so this setting is only for experts.